energy.md

```diff
@@ -414,7 +419,26 @@ Note that the size classes shown are tailored to each region, depending on the d
+GCAM models trade for coal, gas, oil, and bioenergy using an Armington approach that is described below. Under this approach regions are allowed to choose between domestically produced products or globally traded products when making a consumption decision. This approach allows for the computation of a regionally distinct consumption price for fossil fuels based on the domestic and imported consumption. 
+
+Other energy carriers (e.g., solar, wind, geothermal) are not traded. For more generalinformation, see the [discussion of approaches to international trade](trade.html).
+
+### Fossil fuel trade in GCAM
+
+ The figure below depicts the fossil fuel trade structures (using coal as an example). In previous versions of GCAM, every region produced and consumed from a single global market. All crude oil, coal, and natural gas production was sent to a shared market, from which, every region consumed. Only net trade could be tracked and supply was affected by global rather than regional demand. The current structure maintains a global market (e.g. traded coal), but distinguishes between direct consumption of domestic resources and consumption of imported fossil fuels.
+ 
+ <img src="gcam-figs/Fossil_fuel_trade.png" width="900" height="400" /><br/>
+** Schematics of the structures for the flows of the "Coal" commodity in GCAM, with only 3 regions shown for simplicity.**
+{: .fig}
+ 
+#### Data calibration for fossil fuel trade
+
+Each region's share of the global (e.g. traded coal) market as well as the split for domestic and imported goods are calibrated in the final base year. IEA's data set cannot be used to make this calibration because it lacks a bilateral trade accounting. Instead the GCAM data system uses the UN's Comtrade data set to account for intraregional trade to avoid double counting any gross trade. For example trade done within an aggregated GCAM region (e.g. Germany trading with France both of which are in GCAM's EU-15 region) should not be counted as part of that region's gross trade. To avoid overestimating the amounts of gross trade, Comtrade's bilateral data set is used to exclude any intraregional trade. The Comtrade trade data is used to calculate gross trade for each region. This is then combined with the data on production and consumption of fossil fuels calculated within the data system (production is calculated from the fossil fuel supply curves and IEA data and consumption is initialized from IEA energy balances) to compute trade balances. These trade balances are then used to initialize data for the domestic and traded sectors using relevant shareweights and interpolation rules.
+
+
+#### Shareweight, interpolation and logit assumptions
```

Comment:
This seems maybe a bit too in the weeds as (at least right now) this page is really just fairly high level description of the way we model the energy system.  I think we can drop this section for now (I assume it is documented in the core model proposal).

Reply:
Noted. I took this out. The only reasoning behind adding this was to make explicit that we keep domestic demand and exports constant over time while allowing countries to import more if necessary. But, yes, that may be too much for a user anyway. I confirm that this is explained in the CMP.